### DISCUSSION TITLE
When do we release v1.0.0?

### DISCUSSION THREAD

**Discussion by kenjis** [codeigniter4/shield#909] [Polls]
Shield has been released up to beta7. The functionality and documentation have been enhanced considerably.

There is no doubt that the official release of v1.0.0 will be soon.

**Comment by mpmont** [comment]
Yes, I've been using it in a lot of production projects and it has been solid. I think its time.

**Comment by lonnieezell** [comment]
I thought we were ok for a 1.0 release prior to last beta so I definitely vote for an official release. It will boost confidence in the community having a non-beta release out.

**Comment by sammyskills** [comment]
Well, for me, I think the following PRs should be merged before v1.0 release:
1. #908 _(to boost performance)_
2. #867 _(to increase security)_

**Reply by datamweb** [reply]
@sammyskills, we have some open PR that I plan to do before the release.

> 1. [perf: refactor Auth #908](https://github.com/codeigniter4/shield/pull/908) _(to boost performance)_

Regarding this PR, I have already checked, and there is no problem, the reason for not confirming the existence of BC. I said that it may need to be left open for more time so that users may have an opinion. It will definitely be merge before the release of the new version.

**Reply by sammyskills** [reply]
Yeah, that's fine, @datamweb.

Releasing v1.0 right now is fine, but I don't think it will be nice to release v1.0 and update it with BCs within some days or a few week(s), that's why I suggested that the outstanding _(major, with BCs)_ PRs be merged before v1.0.

**Reply by kenjis** [reply]
Yes, if BCs are needed, we would like to do them before the v1.0.0 release.

**Reply by kenjis** [reply]
`supportOldDangerousPassword` has been removed. #976
So all PRs that should be merged seem to be merged.
